src/store/initialState.js

```diff
@@ -0,0 +1,7 @@
+const InitialState = {
```

Comment:
Let the initial state for each of your reducers be in it's file, instead of creating a global initial state object... I guess you want to reset all state on logout, but we'll only be clearing the auth state so other stuff still persist when the user logs in back.